modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -17,7 +21,7 @@
 
     protected String externalZookeeperConnect = null;
 
-    protected SocatContainer proxy;
+    private int port = -1;
```

Comment:
Small nitpick, but what about adding a constant for magic number -1, like `PORT_NOT_INITIALIZED`?
Make the check a bit nicer too.